### ISSUE TITLE
Port forwarding issue with ALCL G-010S-P Stick + map-e-nuro.sh on NURO Hikari 2Gbps

### ISSUE THREAD

**Issue by missing233** [site-u2023/config-software#4]
I'm using ALCL G-010S-P Stick and OpenWrt 23.05.2 with the map-e-nuro.sh script. While IPv6 works fine, I can't get port forwarding to work properly with the IPv4 port that's calculated.
I’ve set the SN and PVID on G-010S-P, and OpenWrt is able to obtain an IPv6 PD. 
After running the map-e-nuro.sh script, I can also get an IPv4 connection. However, no matter how I configure the firewall rules or port forwarding, I’m unable to access specific ports on my local network from the outside.

Has anyone else experienced this issue, or am I missing something in the setup?

**Comment by site-u2023** [comment]
こんにちは
以下、試しましたか？
[ マルチセッション対応（ニチバン対策）]

{
cp /lib/netifd/proto/map.sh /lib/netifd/proto/map.sh.old
wget --no-check-certificate -O /lib/netifd/proto/map.sh https://raw.githubusercontent.com/site-u2023/map-e/main/map.sh.new
read -p " 何かキーを押してデバイスを再起動してください"
reboot
return 0
}

**Comment by missing233** [comment]
> こんにちは
> 
> 以下、試しましたか？
> 
> [ マルチセッション対応（ニチバン対策）]
> 
> 
> 
> {
> 
> cp /lib/netifd/proto/map.sh /lib/netifd/proto/map.sh.old
> 
> wget --no-check-certificate -O /lib/netifd/proto/map.sh https://raw.githubusercontent.com/site-u2023/map-e/main/map.sh.new
> 
> read -p " 何かキーを押してデバイスを再起動してください"
> 
> reboot
> 
> return 0
> 
> }
> 
> 
> 
> 

Yes, I tried replacing map.sh, but the port forwarding still isn't working.

**Comment by site-u2023** [comment]
私はNURO光の環境が無いので検証出来ません。
以下、参考にしてください。

https://blog.hinaloe.net/2024/03/28/nsd-g1000t-map-e-port-exhaustion/

**Comment by leeinsu** [comment]
> I'm using ALCL G-010S-P Stick and OpenWrt 23.05.2 with the map-e-nuro.sh script. While IPv6 works fine, I can't get port forwarding to work properly with the IPv4 port that's calculated. I’ve set the SN and PVID on G-010S-P, and OpenWrt is able to obtain an IPv6 PD. After running the map-e-nuro.sh script, I can also get an IPv4 connection. However, no matter how I configure the firewall rules or port forwarding, I’m unable to access specific ports on my local network from the outside.
> 
> Has anyone else experienced this issue, or am I missing something in the setup?

maybe, BR server address is wrong.
but I don't know how to find BR address..

**Comment by missing233** [comment]
> > I'm using ALCL G-010S-P Stick and OpenWrt 23.05.2 with the map-e-nuro.sh script. While IPv6 works fine, I can't get port forwarding to work properly with the IPv4 port that's calculated. I’ve set the SN and PVID on G-010S-P, and OpenWrt is able to obtain an IPv6 PD. After running the map-e-nuro.sh script, I can also get an IPv4 connection. However, no matter how I configure the firewall rules or port forwarding, I’m unable to access specific ports on my local network from the outside.
> > Has anyone else experienced this issue, or am I missing something in the setup?
> 
> maybe, BR server address is wrong. but I don't know how to find BR address..

Not likely. If the BR server messes up, we’d lose IPv4 internet access altogether.

**Comment by site-u2023** [comment]
close #7
